src/System.Memory/tests/Memory/Span.cs

```diff
@@ -41,6 +41,7 @@ public static void SpanFromCtorArrayLong()
             owner.AsMemory.Span.Validate(91, -92, 93, 94, -95);
         }
 
+        [ActiveIssue(20884, TargetFrameworkMonikers.UapAot)]
```

Comment:
Why are you linking it to a System.Drawing issue? https://github.com/dotnet/corefx/issues/20884

Reply:
Oops. Forgot to hit save :)

Reply:
:)